ambientweather2sqlite/database.py

```
@@ -178,140 +92,268 @@
         for row in cursor:
             result[int(row["hour"])] = dict(row)
 
         return result
+
+# Global database manager instance
+db_manager: DatabaseManager | None = None
+
+
+def get_db_manager(db_path: str | None = None) -> DatabaseManager:
+    """Get the global database manager instance."""
+    global db_manager
```

Ruff (0.11.9)

Using the global statement to update `db_manager` is discouraged

(PLW0603)